errext/exitcodes/codes.go

```diff
@@ -14,11 +14,10 @@ const (
 	SetupTimeout             ExitCode = 100
 	TeardownTimeout          ExitCode = 101
 	GenericTimeout           ExitCode = 102 // TODO: remove?
-	GenericEngine            ExitCode = 103 // TODO: remove after https://github.com/grafana/k6/issues/2804
+	CannotStartRESTAPI       ExitCode = 103
 	InvalidConfig            ExitCode = 104
 	ExternalAbort            ExitCode = 105
-	CannotStartRESTAPI       ExitCode = 106
+	ScriptStoppedFromRESTAPI ExitCode = 106
```

Comment:
Should it be `ScriptAbortedByRESTAPI`?

Reply:
maybe, `ScriptStoppedFromRESTAPI` doesn't "sound" quite right, though `ScriptAbortedByRESTAPI` also doesn't seem correct - it is the user that stops the script, the REST API is just the means to do it :thinking: any other ideas?

Reply:
> it is the user that stops the script, the REST API is just the means to do it
At this point, is it required to have a dedicated exit code? Could we use ExternalAbort?

Reply:
It's not required, but I don't see why we shouldn't have a dedicated exit cod. Since it's so easy to make the differentiation, it doesn't hurt to have it - maybe it matters for some user, or for us in `k6 cloud`, maybe it helps us with debugging some complex issue :man_shrugging: